docs/upgrade-to-7.md

```diff
@@ -93,34 +93,125 @@ Add setup script to create hook. [(#672)]
+
+```diff
+// gasket.config.js
+
+module.exports = {
+  plugins: {
+    add: [
+-      '@gasket/plugin-config'
++      '@gasket/plugin-response-data'
+    ]
+  }
+}
+```
+
+The previous lifecycles in `@gasket/plugin-config` have been renamed.
+
+```diff
+- // /lifecycles/appEnvConfig
++ // /lifecycles/gasketData
+
+- // /lifecycles/appRequestConfig
++ // /lifecycles/responseData
+```
+
 - `gasketData` can be used to tune up the base object during app `init`.
 - `responseData` can be used to tune up the "public" data uniquely for each response.
 
+Environment file configuration has been updated to use `gasketData.dir` instead of `configPath`.
+
+```diff
+// gasket.config.js
+
+module.exports = {
+-  configPath: './src/config'
++  gasketData: {
++    dir: './src/gasket-data'
++  }
+};
+```
+
+The `app.config.js` file has been renamed to `gasket-data.config.js`
+
+```diff
+- <app-root-dir>/app.config.js
++ <app-root-dir>/gasket-data.config.js
+```
+
 [(#680)]
 
 ## Bring Your Own Logger
 
-Gasket's logging infrastructure comprises two main parts:
+Gasket's logging infrastructure was comprised of two main parts:
 
 1. `@gasket/plugin-log`: Manages lifecycle timing and executes the `logTransports` hook for adding extra transports to the logger configuration.
 2. `@gasket/log`: Implements logging using Diagnostics (Client-side) and Winston (Server-side).
 
-While these components work together to initialize `gasket.logger`, not all applications utilize them. Additionally, despite Winston's prevalence, we need to be aware of the numerous logging libraries in the ecosystem when adopting Gasket.
+While these components worked together to initialize `gasket.logger`, not all applications utilized them. Additionally, despite Winston's prevalence, we need to be aware of the numerous logging libraries in the ecosystem when adopting Gasket. The following updates aim to address these concerns:
 
 Updates:
+- Removed `@gasket/log`
+- Created a new `@gasket/plugin-logger` to replace `@gasket/plugin-log`
+- The new logger is included by default in Gasket apps
+- Created `@gasket/plugin-winston` to customize the default logger
+- Added a per-request logger with updateable metadata
+- Updated the default redux logger to use the per-request logger
+- Updated presets to use the winston logger
+
+`@gasket/plugin-winston` is the new default logger for Gasket apps. Use this plugin in place of `@gasket/plugin-log` to customize the default logger.
+
+```diff
+// gasket.config.js
+
+module.exports = {
+  plugins: {
+    add: [
+-      '@gasket/plugin-log'
++      '@gasket/plugin-winston'
+    ]
+  }
+}
+```
+
+`@gasket/plugin-winston` does not support `log` when customizing the logger in `gasket.config.js`.
+
+```diff
+// gasket.config.js
+
+module.exports = {
+-  log: {
+-    prefix: 'my-app'
+-  },
+  winston: {
+    level: 'warning'
+  },
+```
+
+Existing Gasket apps will need to make changes to how they handle logging.
```

Comment:
Maybe add some color here about the default available log levels now follow those of `console`.